.vscode/settings.json

```diff
@@ -1,22 +1,22 @@
 // Place your settings in this file to overwrite default and user settings.
 {
-  "python.pythonPath": "~/.pyenv/shims/python",
-  "editor.formatOnSave": true,
-  "files.exclude": {
-    "**/*.pyc": true,
-    "docs/_build": true,
-    "build": true
-  },
-  "git.ignoreLimitWarning": true,
-  "python.linting.pylintEnabled": true,
-  "python.linting.enabled": true,
-  "python.testing.pytestEnabled": true,
-  "python.formatting.provider": "autopep8",
-  "python.formatting.autopep8Args": [
-      "--ignore-local-config",
-      "--ignore",
-      "E265,E402,E202,E225,E226",
-      "--max-line-length",
-      "120"
-  ]
+    "editor.formatOnSave": true,
+    "files.exclude": {
+        "**/*.pyc": true,
+        "docs/_build": true,
+        "build": true
+    },
+    "git.ignoreLimitWarning": true,
+    "python.linting.pylintEnabled": true,
+    "python.linting.enabled": true,
+    "python.testing.pytestEnabled": true,
+    "python.formatting.provider": "autopep8",
+    "python.formatting.autopep8Args": [
+        "--ignore-local-config",
+        "--ignore",
+        "E265,E402,E202,E225,E226",
+        "--max-line-length",
+        "120"
+    ],
+    "python.pythonPath": "/Users/davidjackson/.pyenv/versions/3.6.8/bin/python"
```

Comment:
revert this file please.

Reply:
done -- sorry about that.

The change I intended to make (and still think we should, if I can figure out how to do it) is to allow local-env overriding of the python path. This settings file is super high-priority, so it takes precedence over everything else.